modules/core/src/main/java/org/apache/ignite/configuration/CacheConfiguration.java

```diff
@@ -427,15 +427,17 @@ public class CacheConfiguration<K, V> extends MutableConfiguration<K, V> {
      */
     private EntryCompressionConfiguration entryCompressionCfg;
 
-    /** Empty constructor (all values are initialized to their defaults). */
+    /** Constructor (all values are initialized to their defaults). */
     public CacheConfiguration() {
-        /* No-op. */
+        this.isStatisticsEnabled = true;
```

Comment:
I think we need to introduce a new Ignite system property (for example, `IGNITE_TCK_COMPLIANCE` or `IGNITE_JSR_107_COMPLIANCE`) that quals to `false` by default. It will allow us to pass the TCK.

Reply:
Fixed. I've created IGNITE_JCACHE_COMPLIANCE property.